src/services/notification.service.ts

```diff
@@ -9,12 +9,16 @@ interface InsertNotificationResult{
   notification_id: number;
 }
 
-const createNotification = async (notification: Notification): Promise<number> => {
+const createNotification = async (notification: Notification): Promise<unknown> => {
   const result = await db.sql<InsertNotificationResult>('insert-notification', {
     to_user_id: notification.toUserId,
     notification_type: notification.notificationType,
   });
-  return result.rows[0].notification_id;
+  const jsonResult = {
+    notificationId: result.rows[0].notification_id,
+    message: 'Successfully created a notification',
+  };
+  return jsonResult;
```

Comment:
This makes sense to me!  You don't necessarily need the `message` item, but it doesn't hurt at all.